### PR TITLE
Oled screen to display dynamic message (Armed and connected)

### PR DESCRIPTION
**Pull request by SunjunKim**
The connection state is updated on the OLED display, in the same manner as the Lua script.
https://cdn.discordapp.com/attachments/744445841779064863/941673285177991229/IMG_9876.mov

# Improvements

* The screen immediately goes back to idle if armed during any onscreen operations.
* On the top line of a screen, some messages and working status could be displayed.
* `[   Connected   ]` is displayed when an RX is get connected (more precisely telemetry is incoming)
* `[   ! Armed !   ]` is displayed when AUX1 is high
~~* `[ Disarm Plz! ]` is displayed for 5s when attempting a button control while armed.~~

AXIS Thor screen is unchanged now because I don't have it – I need your feedback!

### CODE REVIEW

**Comment by SunjunKim** [comment]
Known issue: the connection state is updated properly most of the time, but some glitchy cases are there:

* Normal:  [C] => [-]
* Case 1: [C] => [-] => [C] (brief) => [-]
* Case 2: [C] => [-] => [C]  (stays at this stage)

**Comment by CapnBry** [comment]
I changed the ELRS icon to be a bitmap for TFT LCD so now it supports transparency, as well as removed the degree pixmap and instead use the degree character. I also fixed the wifi update screen which was timing out after 20 seconds back to the idle screen (but staying in wifi mode).

File: src/lib/SCREEN/OLED/oledscreen.cpp [hunk 1]
```diff
@@ -113,18 +114,17 @@ static void displayLogo()
...
+// {
+//     u8g2.clearBuffer();
+//     u8g2.setFont(u8g2_font_profont10_mr);
+// #ifdef USE_OLED_SPI_SMALL
+//     u8g2.drawStr(32, 32, info);
+// #else
+//     u8g2.drawStr(64, 64, info);
+// #endif
+//     u8g2.sendBuffer();
+// }
```
Comment (pkendall64) [inline src/lib/SCREEN/OLED/oledscreen.cpp:127]: Lets just delete it, we have history 😄

File: src/lib/SCREEN/OLED/oledscreen.cpp [hunk 2]
```diff
@@ -425,55 +422,19 @@ void OLEDScreen::doSmartFanValueSelect(int action)
...
+    // if(current_screen_status == SCREEN_STATUS_IDLE)
+    // {
+    //     char buffer[20];
+    //     strncpy(buffer, version, 6);
+    //     sprintf(buffer+6, " %02d", system_temperature);
+    //     // TODO
+    //     // displayFontCenterWithCelsius(0, SCREEN_X/2, SCREEN_LARGE_ICON_SIZE + (SCREEN_Y - SCREEN_LARGE_ICON_SIZE - SCREEN_SMALL_FONT_SIZE)/2,
+    //     //                     SCREEN_SMALL_FONT_SIZE, SCREEN_SMALL_FONT,
+    //     //                     String(buffer), TFT_WHITE,  COLOR_ELRS_BANNER_BACKGROUND);
+    // }
```
Comment (pkendall64) [inline src/lib/SCREEN/OLED/oledscreen.cpp:437]: Delete, we have history

**Review by maybenikhil — APPROVED** [review]
Works as expected on my axis thor TX, and namimno flash OLED TX

**Comment by pkendall64** [comment]
If this is no longer draft shall we remove the label?

**Comment by SunjunKim** [comment]
> If this is no longer draft shall we remove the label?

If it's working on Axis then I draft label could be removed

**Review by pkendall64 — APPROVED** [review]

**Review by CapnBry — APPROVED** [review]

**Pull request merged**